spec/features/user_can_view_admin_dashboard_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+feature "admin dashboard" do
+
+  it "a guest can not view the admin dashboard" do
+    visit '/admin/dashboard'
+    expect(page).to have_content("404")
+  end
+
+  it "a non admin can not view the admin dashboard" do
+    visit '/'
+    click_link "Login"
+    fill_in("Email",  with: "jeff@gmail.com")
+    fill_in("Password",  with: "pass")
+    visit '/admin/dashboard'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.